crates/meroctl/src/cli/app.rs

```diff
@@ -10,7 +11,20 @@ mod get;
 mod install;
 mod list;
 
+pub const EXAMPLES: &str = r"
+  # List all applications
+  $ meroctl -- --home data --node-name node1 application ls
```

Comment:
I think we should skip `--home data` in all these examples, and let it use the default